src/ace/managers/key.c

```diff
@@ -52,8 +54,8 @@ void keyCreate(void) {
 	g_sKeyManager.pInt->is_Node.ln_Type = NT_INTERRUPT;
 	g_sKeyManager.pInt->is_Node.ln_Pri = -60;
 	g_sKeyManager.pInt->is_Node.ln_Name = "ACE_Keyboard_CIA";
-	g_sKeyManager.pInt->is_Data = (APTR)&g_sKeyManager;
-	g_sKeyManager.pInt->is_Code = keyIntServer;
+	g_sKeyManager.pInt->is_Data = 0;
+	g_sKeyManager.pInt->is_Code = (void(*)(void))keyIntServer;
```

Comment:
What's the background of this change?

Reply:
Bebbo GCC's OS headers had at some point wrong interrupt struct decls, which were causing warnings for interrupt servers with custom argument list. Cast is not needed now. Fixed in 4296300.

Regarding data, this field contains value which is passed as interrupt handler's arg. Not needed here since manager is global and inside handler there is a function call which internally uses global manager.